.github/ISSUE_TEMPLATE/game_issue.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug report about game issues
+about: This report type is speficically meant or issues with games
+
+---
+
+**Game information**
+<!-- Name of the game or games that do not work -->
+
+**Distribution name**
+<!-- If your distribution is versioned, incldue the version. Also preferably last update time -->
```

Comment:
Perhaps include a command to get the information? `lsb_release --all`?

Reply:
I have no idea if there's a single way that works everywhere

Reply:
It is Linux, of course there is no way that works everywhere!

`cat /etc/redhat-release || cat /etc/suse-release || cat /etc/debian-release || cat /etc/lsb_release || cat /etc/os-release`

Who knows.